packages/server-ct/src/project-ct.ts

```diff
@@ -304,7 +303,7 @@ export default class Project {
         this.emit('socket:connected', id)
       },
 
-      onSetRunnables (runnables: Runnables) {
+      onSetRunnables (runnables: RootRunnable) {
```

Comment:
thanks, how did you figure out these types?

Reply:
When I added a tsconfig.json, the file started to be all red...
I followed the path of the F12.